tests/tests_app/core/test_lightning_app.py

```diff
@@ -482,6 +482,21 @@ def make_delta(i):
         assert generated > expect
 
 
+def test_lightning_app_aggregation_empty():
+    """Verify the app breaks if no delta is found."""
```

Comment:
Can you elaborate? Why should we break if no deltas are found, shouldn't be resilient here?

Reply:
My bad, terrible description ...
`Verify the while loop exits before `state_accumulate_wait` is reached if no deltas are found.`